### PR TITLE
fix: make run-kube-local failed

### PR DESCRIPTION
**Pull request by Xunzhuo**
Fixes: #387
Signed-off-by: Xunzhuo <bitliu@tencent.com>

### CODE REVIEW

File: tools/make/kube.mk [hunk 1]
```diff
@@ -43,7 +43,7 @@ kube-undeploy: kube-uninstall ## Uninstall the Envoy Gateway controller into the
 	$(tools/kustomize) build internal/provider/kubernetes/config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f - 
 
 .PHONY: run-kube-local ## Run EG locally.
-run-kube-local: kube-install
+run-kube-local: build kube-install
```
Comment (arkodg) [inline tools/make/kube.mk:46]: while you are here can you also move `## Run EG locally.` to this line, so it shows up in `make help`, TIA

Comment (Xunzhuo) [reply tools/make/kube.mk]: Done.

**Review by arkodg — APPROVED** [review]
🚀

**Pull request merged**